### PR TITLE
last-minute renames

### PR DESCRIPTION
**Pull request by phryneas**
`entity` -> `tag`
`entityTypes` -> `tagTypes`
`provides` -> `providesTags`
`invalidates` -> `invalidatesTags`
`prefetchThunk` -> `prefetch`

### CODE REVIEW

File: docs/concepts/mutations.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ Notice the `onStart`, `onSuccess`, `onError` methods? Be sure to check out how t
...
   BaseQuery extends BaseQueryFn,
-  EntityTypes extends string,
+  TagTypes extends string,
   ResultType,
   ReducerPath extends string = string,
   Context = Record<string, any>
 > = BaseEndpointDefinition<QueryArg, BaseQuery, ResultType> & {
   type: DefinitionType.mutation;
-  invalidates?: ResultDescription<EntityTypes, ResultType, QueryArg>;
+  invalidates?: ResultDescription<TagTypes, ResultType, QueryArg>;
```
Comment (msutkowski) [inline docs/concepts/mutations.md:51]:
```suggestion
  invalidatesTags?: ResultDescription<TagTypes, ResultType, QueryArg>;
```

File: docs/concepts/mutations.md [hunk 2]
```diff
@@ -42,13 +42,13 @@ Notice the `onStart`, `onSuccess`, `onError` methods? Be sure to check out how t
...
   provides?: never;
```
Comment (msutkowski) [inline docs/concepts/mutations.md:52]:
```suggestion
  providesTags?: never;
```

File: docs/concepts/mutations.md [hunk 3]
```diff
@@ -119,17 +119,17 @@ export const PostDetail = () => {
...
 
 #### Provides
 
-A _query_ can _provide_ entities to the cache. The `provides` argument can either be an array of `string` (such as `['Posts']`), `{type: string, id?: string|number}` or a callback that returns such an array. That function will be passed the result as the first argument, the response error as the second argument, and the argument originally passed into the `query` method as the third argument. Note that either the result or error arguments may be undefined based on whether the query was successful or not.
+A _query_ can _provide_ tags to the cache. The `provides` argument can either be an array of `string` (such as `['Posts']`), `{type: string, id?: string|number}` or a callback that returns such an array. That function will be passed the result as the first argument, the response error as the second argument, and the argument originally passed into the `query` method as the third argument. Note that either the result or error arguments may be undefined based on whether the query was successful or not.
```
Comment (msutkowski) [inline docs/concepts/mutations.md:128]:
```suggestion
A _query_ can _provide_ tags to the cache. The `providesTags` argument can either be an array of `string` (such as `['Posts']`), `{type: string, id?: string|number}` or a callback that returns such an array. That function will be passed the result as the first argument, the response error as the second argument, and the argument originally passed into the `query` method as the third argument. Note that either the result or error arguments may be undefined based on whether the query was successful or not.
```

File: docs/concepts/mutations.md [hunk 4]
```diff
@@ -119,17 +119,17 @@ export const PostDetail = () => {
...
 
 #### Invalidates
 
-A _mutation_ can _invalidate_ specific entities in the cache. The `invalidates` argument can either be an array of `string` (such as `['Posts']`), `{type: string, id?: string|number}` or a callback that returns such an array. That function will be passed the result as the first argument, the response error as the second argument, and the argument originally passed into the `query` method as the third argument. Note that either the result or error arguments may be undefined based on whether the mutation was successful or not.
+A _mutation_ can _invalidate_ specific tags in the cache. The `invalidates` argument can either be an array of `string` (such as `['Posts']`), `{type: string, id?: string|number}` or a callback that returns such an array. That function will be passed the result as the first argument, the response error as the second argument, and the argument originally passed into the `query` method as the third argument. Note that either the result or error arguments may be undefined based on whether the mutation was successful or not.
```
Comment (msutkowski) [inline docs/concepts/mutations.md:132]:
```suggestion
A _mutation_ can _invalidate_ specific tags in the cache. The `invalidatesTags` argument can either be an array of `string` (such as `['Posts']`), `{type: string, id?: string|number}` or a callback that returns such an array. That function will be passed the result as the first argument, the response error as the second argument, and the argument originally passed into the `query` method as the third argument. Note that either the result or error arguments may be undefined based on whether the mutation was successful or not.
```

**Review by msutkowski — COMMENTED** [review]
Should these be updated?

**Comment by phryneas** [comment]
Yup, good catch!

**Review by msutkowski — APPROVED** [review]
Nice work! This is... what is the word the hip folk say... **_chonky_**? :trophy:

**Review by markerikson — APPROVED** [review]
Looks good to me as well.

**Pull request merged**